### PR TITLE
Feat/gallery

### PR DESCRIPTION
**Pull request by elifalinli**

### CODE REVIEW

File: app/_components/Gallery.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <h2 className="text-3xl font-bold sm:text-4xl">
+          Our <span className=" text-primary">Gallery</span>
+        </h2>
+        <p className="mt-4 text-gray-600">
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Consequuntur
+          aliquam doloribus nesciunt eos fugiat. Vitae aperiam fugit
+          consequuntur saepe laborum.
+        </p>
+      </div>
+
```
Comment (muhsinaltintop) [inline app/_components/Gallery.jsx:25]: unnecessary space

File: app/_components/Gallery.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            dragFree: false,
+          }}
+        >
+          <CarouselContent className="-ml-1">
+            {data.map((item, index) => {
+              return (
+                <CarouselItem
+                  key={index}
+                  className="md:basis-2/3 lg:basis-8/10 xl:basis-9/10 px-2"
+                >
```
Comment (muhsinaltintop) [inline app/_components/Gallery.jsx:39]: unnecessary enter

File: components/ui/card.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import * as React from "react"
+
```
Comment (muhsinaltintop) [inline components/ui/card.jsx:2]: unnecessary enter

File: components/ui/carousel.jsx [hunk 4]
```diff
@@ -0,0 +1,194 @@
+"use client";
+import * as React from "react"
+import useEmblaCarousel from "embla-carousel-react";
+import { ArrowLeft, ArrowRight } from "lucide-react"
+
```
Comment (muhsinaltintop) [inline components/ui/carousel.jsx:5]: unnecessary enter

File: components/ui/carousel.jsx [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+      event.preventDefault()
+      scrollNext()
+    }
+  }, [scrollPrev, scrollNext])
+
+  React.useEffect(() => {
+    if (!api || !setApi) {
+      return
+    }
+
```
Comment (muhsinaltintop) [inline components/ui/carousel.jsx:71]: unnecessary enter

File: components/ui/carousel.jsx [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+    setCanScrollPrev(api.canScrollPrev())
+    setCanScrollNext(api.canScrollNext())
+  }, [])
+
+  const scrollPrev = React.useCallback(() => {
+    api?.scrollPrev()
+  }, [api])
+
```
Comment (muhsinaltintop) [inline components/ui/carousel.jsx:52]: unnecessary enter

File: components/ui/carousel.jsx [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+    }
+
+    setApi(api)
+  }, [api, setApi])
+
+  React.useEffect(() => {
+    if (!api) {
+      return
+    }
+
```
Comment (muhsinaltintop) [inline components/ui/carousel.jsx:79]: unnecessary enter

**Review by muhsinaltintop — COMMENTED** [review]
There are unnecessary enters and spaces otherwise looks good to me.

**Review by elifalinli — COMMENTED** [review]
removed enters

**Pull request merged**